src/Graphics/tests/TestPathBounds.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please avoid creating new projects. Our CI is already configured with the necessary ones. Instead, use the following path to add new samples: https://github.com/dotnet/maui/tree/58f3837898fd072fe175e8fa95c15b296c50a5f0/src/Graphics/samples/GraphicsTester.Portable/Scenarios